docs/tutorial/part-zero/index.md

```diff
@@ -1,325 +1,330 @@
 ---
-title: Set Up Your Development Environment
+title: Pystytä sinun kehitysympäristö
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Before you start building your first Gatsby site, you’ll need to familiarize yourself with some core web technologies and make sure that you have installed all required software tools.
+Ennen kuin aloitat sinun ensimmäisen Gatsby-sivuston rakentamisen, sinun on perehdyttävä joihinkin ydin web-teknologioihin ja varmistettava, että olet asentanut kaikki vaadittavat ohjelmistotyökalut.
```

Comment:
Let's avoid using wording like `sinun Gatsby-sivusto` and use `Gatsby-sivustosi` and use possessive suffixes. Suggested the same above in the title :)

```suggestion
Ennen kuin aloitat ensimmäisen Gatsby-sivustosi rakentamisen, sinun on perehdyttävä joihinkin ydinweb-teknologioihin ja varmistettava, että olet asentanut kaikki vaadittavat ohjelmistotyökalut.
```

Reply:
Yeah, that's better. `Part-Zero` was quite long to translate, so i'm pretty sure I have used that somewhere else.